api/openapi.yaml

```diff
@@ -18,6 +18,17 @@ tags:
     description: "Get the list of all envoy fleets"
   - name: "static routes"
     description: "Get the list of all static routes"
+x-kusk:
```

Comment:
There will be a templated version of this in the helm chart for the api, im not sure we need it here but i dont think it will hurt

Reply:
I just put it here as a suggestion tbh.. I was curious if I was able to get it to work, so I kept the results here...

I wonder if we can use the dashboard/api as our dogfooding!